### PR TITLE
Update the criteria of selection of F1-F1bis of 4/2015

### PR DESCRIPTION
**Pull request by Xevib**
### Description:

> Update the selection criteria of F1 and F1 bis

### Tasks:

- [x] Update the F1 and F1bis selection criteria
- [x] Update dates

### Links:

> Add links for each file modified

* [4/2015](http://builds.gisce.net/powerp-docs/powerp_updt_42015_f1/ca/distri/adm-pub/resoluciones/circular_4-2015/)
* [4/2015](http://builds.gisce.net/powerp-docs/powerp_updt_42015_f1/es/distri/adm-pub/resoluciones/circular_4-2015/)

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_updt_42015_f1/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_updt_42015_f1/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_updt_42015_f1/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_updt_42015_f1/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_updt_42015_f1/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/distri/adm-pub/resoluciones/circular_4-2015.md [hunk 1]
```diff
@@ -115,14 +110,15 @@ S'inclourà informació relativa dels CUPS que compleixin les següents
...
-* **Data de creació**: La data de creació del **CUPS** ha de ser inferior al 01/01/2015 o estar buida.
+* **Data de creació**: La data de creació del **CUPS** ha de ser inferior al 01/01 del any del inventari de les dades o estar buida.
 
 També s'inclourà informació de les **pòlisses** dels CUPS que compleixin les
 següents característiques:
 
-* **Data d'alta de la pòlissa**: Ha de ser més petita o igual que l'últim dia de l'any (31/12/2014).
-* **Data de baixa de la pòlissa**: Ha de ser superior a l'ultim dia de l'any (31/12/2014) o bé ha
+* **Data d'alta de la pòlissa**: Ha de ser més petita o igual que l'últim dia de l'any (31/12) .
+* **Data de baixa de la pòlissa**: Ha de ser inferior a l'ultim dia de l'any (31/12) de l'any d'inventari de les dades o bé ha
```
Comment (jaumef) [inline docs/distri/adm-pub/resoluciones/circular_4-2015.md]: "o bé ha pot estar buida" → "o bé pot estar buida"

Comment (jaumef) [inline docs/distri/adm-pub/resoluciones/circular_4-2015.md]: ES: `Fecha de bajan` → `Fecha de baja`

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_updt_42015_f1/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_updt_42015_f1/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Pull request merged**